tests/test_speech_to_text.py

```diff
@@ -1,25 +1,66 @@
+import os
+import boto3
 import json
 from pathlib import Path
 
-import dotenv
+import moto
 import pytest
 import speech_to_text
 
-# set AWS_PROFILE from .env in the environment
-dotenv.load_dotenv()
+BUCKET = "bucket"
+TODO_QUEUE = "todo"
+DONE_QUEUE = "done"
+
+
+@pytest.fixture(scope="function")
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+    os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
+    os.environ["SPEECH_TO_TEXT_TODO_SQS_QUEUE"] = TODO_QUEUE
+    os.environ["SPEECH_TO_TEXT_DONE_SQS_QUEUE"] = DONE_QUEUE
+    os.environ["SPEECH_TO_TEXT_S3_BUCKET"] = BUCKET
+
+
+@pytest.fixture(scope="function")
+def sts(aws_credentials):
+    with moto.mock_aws():
+        yield boto3.client("sts")
+
+
+@pytest.fixture(scope="function")
+def s3(aws_credentials):
+    with moto.mock_aws():
+        yield boto3.client("s3")
+
+
+@pytest.fixture(scope="function")
+def sqs(aws_credentials):
+    with moto.mock_aws():
+        yield boto3.resource("sqs")
+
+
+@pytest.fixture(scope="function")
+def bucket(s3):
+    return s3.create_bucket(Bucket=BUCKET)
+
+
+@pytest.fixture(scope="function")
+def queues(sqs):
+    sqs.create_queue(QueueName=TODO_QUEUE)
+    sqs.create_queue(QueueName=DONE_QUEUE)
```

Comment:
All of the above is for mocking out STS, SQS and S3 services with [moto](https://docs.getmoto.org/en/latest/). moto allows these mocked out services to operate as if they were actually running, but with fake local resources.